<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MEConfig</class>
 <widget class="QDialog" name="MEConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>537</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MEPIS X-Windows Assistant</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="repairTabPage">
      <attribute name="icon">
       <iconset>
        <normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/apps/package_development.png</normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/apps/package_development.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Repair</string>
      </attribute>
      <attribute name="toolTip">
       <string>Repair an X installation</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="reinstallGroupBox">
         <property name="title">
          <string>Copy Current XConfig</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="1">
           <widget class="QComboBox" name="rootCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select partition containing root</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="diskCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select disk to copy to</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="rootLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font/>
            </property>
            <property name="toolTip">
             <string>Partition containing root</string>
            </property>
            <property name="text">
             <string>/ (root) partition:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="useDiskLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font/>
            </property>
            <property name="toolTip">
             <string>Disk to copy to</string>
            </property>
            <property name="text">
             <string>Destination disk:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>261</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="generalTabPage">
      <attribute name="icon">
       <iconset>
        <normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/apps/fonts.png</normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/apps/fonts.png</iconset>
      </attribute>
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <attribute name="toolTip">
       <string>General X configs</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="sizeGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Relative Size of Text</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="2">
           <widget class="QRadioButton" name="smallerCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select for small text</string>
            </property>
            <property name="text">
             <string>Small (75 dpi)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="mediumCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select for medium text</string>
            </property>
            <property name="text">
             <string>Medium (96 dpi)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="largerCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select for large text</string>
            </property>
            <property name="text">
             <string>Large (120 dpi)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonAbout">
       <property name="toolTip">
        <string>About this app</string>
       </property>
       <property name="text">
        <string>About...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/help.png</normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/help.png</iconset>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generalHelpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/apps/khelpcenter.png</normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/apps/khelpcenter.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>116</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../../../share/icons/mepis.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>75</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="toolTip">
        <string>Apply any changes and quit</string>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/button_ok.png</normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/button_ok.png</iconset>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="toolTip">
        <string>Apply any changes</string>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/configure.png</normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/configure.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="toolTip">
        <string>Cancel changes and quit</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/button_cancel.png</normaloff>../../../../../../share/icons/nuvola_platin_me/16x16/actions/button_cancel.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonAbout</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonApply</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>diskCombo</tabstop>
  <tabstop>rootCombo</tabstop>
  <tabstop>largerCheck</tabstop>
  <tabstop>mediumCheck</tabstop>
  <tabstop>smallerCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
